README.md

```diff
@@ -2,202 +2,238 @@
 
-All you need to start using the VTEX platform.
+The CLI that offers all you need to start using the VTEX IO platform efficiently.
+
+#### Summary
+- [How do we empower you?](#how-do-we-empower-you)
+- [First step: Installing the VTEX Toolbelt](#installing-the-toolbelt-with-yarn)
+- [Developing an app locally](#to-develop-locally-use-vtex-link)
+- [Customize your prompt](#wanna-know-something-cool-you-can-customize-your-prompt)
+- [FAQ](#frequently-asked-questions)
+- [Contributing](#contributing)
+- [Send us feedback](#tell-us-what-you-think-about-toolbelt)
+
+# How do we empower you? 
+
+VTEX Toolbelt is a command-line program that gives you access to all of 
+the features in the VTEX platform and increases your productivity, allowing you to:
+
+- **Manage your apps quickly:** List, install and uninstall available apps in your account;
+- **Work safely on your personal environments:** List, create and remove workspaces;
+- **Easily ship changes to your final user:** Promote a workspace to `master`, making it public in your account URL;
+- **See what you are creating in real time:** Develop new apps in link mode, using automatic synchronization;
+- **Go live from the command line:** Publish apps to the VTEX App Store.
+
+# Getting started: Installing the VTEX Toolbelt
+To use the VTEX Toolbelt, you'll need to install other engines 
+that will also be useful when developing for VTEX.
+
+#### Node.js to run the tool
+- Install Node.js [here](https://nodejs.org/en/download/): Toolbelt is written in Typescript and designed to run on Node.js. 
+(Node is the runtime that executes the JS and allows our tool to run and work at your computer) 
+- Install yarn [here](https://classic.yarnpkg.com/en/docs/install): yarn is a way to manage your code, organize content in packages 
+that you can download, share and... install, just like our CLI :) 
+
+Already has Node.js and Yarn? Let's install the Toolbelt. 
+
+#### Installing the Toolbelt with yarn
+`yarn` powers app development to React, Node and VTEX IO! Install the Toolbelt using yarn and start working with us generating automatically all the files you'll need to create apps in the future. 
+To install it, run: 
+ 
+ ```
+ yarn global add vtex
+ ```
+
+<details>
+  <summary>Alternative installation method using npm</summary>
+  You can install using `npm`. But remember, if in the future 
+  you want to develop a VTEX IO app, you'll need to install `yarn` 
+
+  To install with `npm`, run:
+
+  ```
+  npm install -g vtex
+  ```
+</details>
+
+# Using the CLI 
+
+After installation, run `vtex` to see our commands and what each one of them do.
 
-### What is this?
+<details>
+    <summary>Or, check all our commands here</summary>
 
-VTEX Toolbelt is a command-line program that gives you access to all of the features in the VTEX platform.
+  ```
+  $ vtex
```

Comment:
How about we change this to point to this file?
https://github.com/vtex/toolbelt/blob/9e80d10768460a2337effed877910d0b4d5cbcfb/docs/commands.md

Reply:
Adding a link to the file? I don't know if I understood the suggestion :x